fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/MetricLoader.java

```diff
@@ -16,4 +17,13 @@
      */
     void loadMetricDictionary(MetricDictionary metricDictionary);
```

Comment:
Can we deprecate the "old" method, since we want to allow dimensions always anyways?